src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/AddAzureSqlDatabaseToFailoverGroup.cs

```diff
@@ -44,19 +54,11 @@ public class AddAzureSqlDatabaseToFailoverGroup : AzureSqlFailoverGroupCmdletBas
         /// </summary>
         [Parameter(Mandatory = true,
             ValueFromPipeline = true,
-            HelpMessage = "The Azure SQL Database to be added to the secondary server.")]
+            HelpMessage = "One or more Azure SQL Databases on the Failover Group's primary server to be added to the Failover Group.")]
         [ValidateNotNullOrEmpty]
         public List<AzureSqlDatabaseModel> Database { get; set; }
```

Comment:
Sure, but PowerShell beginners do not use piping. Piping is cool, I use it all the time, but I have watched people use PowerShell for the first time and it takes a while to figure this out. In fact most powershell beginner just type the command name `Add-AzureRmSqlDatabaseToFailoverGroup` and then follow the prompts for required parameters. 

This means in order to make your cmdlet approachable there needs to be a param set like `Add-AzureRmSqlDatabaseToFailoverGroup -ServerName myserver -FailoverGroupName mygroup -DatabaseName mydatabase`. Yes it's redundant once you've learned the more advanced piping technique, but we often make redundancies in order to improve UX.

In fact, the piping scenario would work if you had `DatabaseName` parameter with `ValueFromPipelineByPropertyName=true`. So it's actually `DatabaseName` that makes `Database` redundant, not the other way around :)